frontend/app/views/spree/products/_cart_form.html.erb

```diff
@@ -29,7 +29,7 @@
     <% end %>
 
     <% if @product.price_for(current_pricing_options) and !@product.price.nil? %>
-      <div data-hook="product_price" class="columns five <% if !@product.has_variants? %> alpha <% else %> omega <% end %>">
+      <div data-hook="product_price" class="columns five <%= @product.has_variants? ? 'alpha' : 'omega' %>">
```

Comment:
You are right, I thought this was going to be my first PR without comments hahaha, I'll change it ASAP